DQM/L1TMonitor/src/L1TStage2EMTF.cc

```diff
@@ -896,11 +955,22 @@ void L1TStage2EMTF::analyze(const edm::Event& e, const edm::EventSetup& c) {
     if (Hit->Is_GEM() == true) {
       gemHitBX->Fill(Hit->BX(), (endcap > 0) ? 1.5 : 0.5);
       hist_index = (endcap > 0) ? 1 : 0;
+      //Added def of layer
+      int layer = Hit->Layer();
+      int phi_part = Hit->Pad() / 64;  // 0-2
+      int vfat = phi_part * 8 + Hit->Partition();
       if (Hit->Neighbor() == false) {
         gemChamberPad[hist_index]->Fill(chamber, Hit->Pad());
         gemChamberPartition[hist_index]->Fill(chamber, Hit->Partition());
         gemHitOccupancy->Fill(chamber_bin(1, 1, chamber), (endcap > 0) ? 1.5 : 0.5);  // follow CSC convention
-      } else {
+        //Added plots 07-21-22 ***
+        gemVFATBXPerChamber[chamber - 1][hist_index][layer]->Fill(Hit->BX(), vfat);
+        //indexed plots by BX 07-21-22
+        gemChamberVFATBX[hist_index][Hit->BX() + 3]->Fill(chamber_bin(1, 1, chamber), vfat);
```

Comment:
It seems "Hit->BX() + 3" can be as large as "7" and exceeds the boundary, which then cause the issue as reported in https://github.com/cms-sw/cmssw/issues/38940